### DISCUSSION TITLE
Нет метрики времени вызова БД.

### DISCUSSION THREAD

**Discussion by Barrya42** [LazarenkoA/prometheus_1C_exporter#47] [Q&A] [answered]
Судя по коду, она должна называться durationcurrentdbms. Но на странице http://localhost:9091/1C_Metrics ее нет и ни разу не было. Остальные метрики на месте.

Еще было бы очень полезно в файле README описать маппинг метрик 
"Имя в консоли 1с" : "Имя в экспортере"
Сейчас их имена не очень очевидны.

**Comment by LazarenkoA** [comment]
`durationcurrentdbms` это `duration-current-dbm` (текущее время вызова СУБД), это не отдельная метрика, это разрез в метрике SessionsData, заполняется если его значение > 0 

про README, согласен, как-нибудь сделаю

**Reply by Barrya42** [reply, marked as answer]
Да, про разрез, не так выразился. Понял почему не отображается, При обновлении метрики раз в 10 сек, она всегда в этот момент равна 0.
